packages/app/components/pack/AddPack.tsx

```diff
@@ -73,7 +73,7 @@ export const AddPack = ({ isCreatingTrip = false, onSuccess }) => {
             label="Name"
             style={{ textAlign: 'left', width: 200 }}
           />
-          {Platform.OS === 'web' ? (
+          {Platform.OS !== 'web' ? (
```

Comment:
Why change the condition?, won't that make the isPublic selector not to show on web?

Reply:
No, that will just change the context menu option to use the context menu.
But you are still right, I shouldn't had to change that and will revert that change on my next commit.
Thank you for pointing it out.